src/main/webapp/app/exercises/programming/manage/programming-exercise-detail.component.ts

```diff
@@ -343,6 +384,7 @@ export class ProgrammingExerciseDetailComponent implements OnInit, OnDestroy {
                         exerciseId: exercise.id,
                         type: 'TEMPLATE',
                         showOpenLink: !this.localVCEnabled,
+                        checkoutDirectory: this.localVCEnabled ? this.checkoutDirectories?.exerciseCheckoutDirectory : undefined,
```

Comment:
If we are not on LocalCI we display the build plans instead (there the checkout directories can be looked up)